lib/fpm/package/deb.rb

```diff
@@ -905,7 +905,7 @@ def mkdir_p(dir)
   def to_s(format=nil)
     # Default format if nil
     # git_1.7.9.3-1_amd64.deb
-    return super("NAME_FULLVERSION_ARCH.TYPE") if format.nil?
+    return super("NAME_FULLVERSION_ARCH.EXTENSION") if format.nil?
```

Comment:
Don't mistake "ruby" with "rude".  :) I needed two iterations to discover why there are two ``return`` instructions one after the another. That was crazy, as I'm not used to read whole lines of code, only first one or two words :)